src/app/admin/Smaregi/Index.vue

```diff
@@ -31,68 +31,97 @@
+              スマレジ登録店舗：{{ shop.storeName }}
+            </div>
+            <div class="pl-4 pt-2">
+              連携する店舗：
+              <o-select
+                v-model="selectedRestaurant[k]"
+                :class="
                 selectedRestaurant[k] && duplicateElement[selectedRestaurant[k]]
                   ? 'border-2 border-solid border-red-700'
                   : ''
               "
-            >
-              <option
-                v-for="restaurant in restaurants"
-                :value="restaurant.id"
-                :key="restaurant.id"
-              >
-                {{ restaurant.restaurantName }}
-              </option>
-            </o-select>
-
-            在庫切れしきい値:
-            <o-select v-model="outOfStockData[k]">
-              <option
-                v-for="threshold in outOfStockThresholds"
-                :value="threshold.value"
-                :key="threshold.value"
               >
-                {{ threshold.name }}
-              </option>
-            </o-select>
-
-            在庫復活しきい値:
-            <o-select v-model="inStockData[k]">
-              <option
-                v-for="threshold in inStockThresholds"
-                :value="threshold.value"
-                :key="threshold.value"
-              >
-                {{ threshold.name }}
-              </option>
-            </o-select>
+                <option
+                  v-for="restaurant in restaurants"
+                  :value="restaurant.id"
+                  :key="restaurant.id"
+                >
+                  {{ restaurant.restaurantName }}
+                </option>
+              </o-select>
+            </div>
+
+            <div class="pl-4 pt-1">
+              在庫切れしきい値:
+              <o-select v-model="outOfStockData[k]">
+                <option
+                  v-for="threshold in outOfStockThresholds"
+                  :value="threshold.value"
+                  :key="threshold.value"
+                >
+                  {{ threshold.name }}
+                </option>
+              </o-select>
+            </div>
+
+            <div class="pl-4 pt-1 pb-4">
+              在庫復活しきい値:
+              <o-select v-model="inStockData[k]">
+                <option
+                  v-for="threshold in inStockThresholds"
+                  :value="threshold.value"
+                  :key="threshold.value"
+                >
+                  {{ threshold.name }}
+                </option>
+              </o-select>
+            </div>
           </div>
           <div v-if="isDuplicateError">*お店の指定が重複しています</div>
-          <o-button @click="saveShops" :disabled="isDuplicateError"
-            >保存</o-button
-          >
+          <div class="mt-4">
+            <button @click="saveShops" :disabled="isDuplicateError">
+              <div
+                class="inline-flex h-12 items-center justify-center rounded-full bg-op-teal px-6 shadow"
+                style="min-width: 8rem"
```

Comment:
```suggestion
                style="min-width: 8rem"
```

style="min-width: 8rem" は tailwindcss に `min-w-32` という class があるので、それを使うのが良いと思います

https://tailwindcss.com/docs/min-width